packages/@react-spectrum/overlays/src/Modal.tsx

```diff
@@ -84,7 +84,11 @@ let ModalWrapper = forwardRef(function (props: ModalWrapperProps, ref: RefObject
   );
 
   let viewport = useViewportSize();
+  // TODO: try getting rid of this reliance on useViewport size in favor of height: "100svh" and see if that makes the
+  // modal. Maybe to do later
   let style: any = {
+    // TODO: look up why we have this and if we can use svh instead so we don't run into a modal with combobox in it + pinch zoom breaking the combobox dropdown placement
+    // not sure if that even happens though
```

Comment:
The behavior of the modal is the same before and after this change (aka it adjusts itself to fit the visual viewport), will need to revisit to see if we want its size to remain static as the user pinch zooms. For now we'll keep the same behavior